src/tasks/containers/TasksPage.test.tsx

```diff
@@ -151,6 +152,7 @@ const setup = (override = {}) => {
         status: RemoteDataState.Done,
       },
       tasks: {
+        ...initialState(),
```

Comment:
needed to add `showInactive` state here so `expect(ui.store.getState().resources.tasks.showInactive).toBe(true)` can pass; line 188